R/g.analyse.perday.R

```diff
@@ -508,7 +547,8 @@ g.analyse.perday = function(ndays, firstmidnighti, time, nfeatures,
                       # METHOD 1: time spent above threhold based on 5 sec epoch
                       mvpa[1] = length(which(varnum * UnitReScale >= params_phyact[["mvpathreshold"]][mvpai])) / (60/ws3) #time spent MVPA in minutes
                       # METHOD 2: time spent above threshold based on 1minute epoch
-                      varnum2 = cumsum(c(0, varnum))
+                      varnum_NA2zero = replace(varnum, which(is.na(varnum)), 0)
```

Comment:
This is done here to allow the aggregation to 1-min and 5-min epochs. `cumsum()` does not work well in a vector containing NAs. Since this is used for MVPA definition, this NA time (redefined as 0) would never be counted as MVPA because it will not go over the `mvpathreshold`. Can you see any side effect of this? I have tried in some files and I always get the expected MVPA time.

Reply:
Addressed in the last commits, NAs are only used in time-based metrics (https://github.com/wadpac/GGIR/pull/709#discussion_r1064134228)